modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -189,6 +197,10 @@ function calculateViewportUniforms({
     // Projection mode values
     project_uCoordinateSystem: shaderCoordinateSystem,
     project_uCenter: projectionCenter,
+    project_uWrapCoordinates: wrapCoordinates,
```

Comment:
nit: `project_uLngLatCenter` 

We only use longitude, not both values, and they are specific to one coordinate system.

I wonder if the uniform could either be:

* minimized to just `project_uMeridian` or similar.
* generalized to not be lng/lat (`project_uUnprojectedCenter`)